Packages/packages-lock.json

```diff
@@ -1,16 +1,38 @@
 {
   "dependencies": {
+    "com.cesium.unity": {
+      "version": "1.14.1",
+      "depth": 0,
+      "source": "registry",
+      "dependencies": {
+        "com.unity.mathematics": "1.2.0",
+        "com.unity.test-framework": "1.1.31",
+        "com.unity.shadergraph": "12.1.6",
+        "com.unity.inputsystem": "1.4.2"
+      },
+      "url": "https://unity.pkg.cesium.com"
+    },
```

Comment:
It looks like this got merged already but I think we usually leave this out because Unity auto-fills this for us (and it gives us less to edit when we make a new release).

Reply:
Mm yeah, this also gets affected when developing with Cesium for Unity (such that the package is embedded instead). I'm going to open a PR that removes this just to keep things clean.